src/BladeError/BladeError.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace HelsingborgStad\BladeError;
+
+use Throwable;
+use Exception;
+use HelsingborgStad\BladeService\BladeService as BladeErrorRenderer;
+
+/**
+ * Class BladeServiceInstance
+ *
+ * This class provides functionality for managing Blade views.
+ */
+class BladeError implements BladeErrorInterface
+{
+  public function __construct(private Throwable $e, private array $viewPaths = [], private array $viewData = []){}
+
+  public function print(): void 
```

Comment:
It would be valuable to have a test for this function. More specifically we would like to ensure that the second parameter of makeView contains expected values.